File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/MyApplication.java
package com.mentalmachines.droidcon_boston;

import android.app.Application;
import com.google.firebase.FirebaseApp;
import com.twitter.sdk.android.Twitter;
import com.twitter.sdk.android.core.TwitterAuthConfig;
import io.fabric.sdk.android.Fabric;

public class MyApplication extends Application {

  @Override
  public void onCreate() {
    super.onCreate();

    TwitterAuthConfig authConfig = new TwitterAuthConfig(getResources().getString(R.string.twitter_key),
        getResources().getString(R.string.twitter_secret));
    Fabric.with(this, new Twitter(authConfig));

    FirebaseApp.initializeApp(this);
  }
}

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/MyApplication.kt
package com.mentalmachines.droidcon_boston

import android.app.Application
import com.jakewharton.threetenabp.AndroidThreeTen
import com.mentalmachines.droidcon_boston.utils.NotificationUtils
import com.twitter.sdk.android.core.Twitter
import timber.log.Timber

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        Twitter.initialize(this)

        AndroidThreeTen.init(this)

        NotificationUtils(this).scheduleMySessionNotifications()

        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
    }
}
